src/auslib/web/public/json.py

```diff
@@ -1,11 +1,36 @@
 import json
 
+import semver
 from flask import Response
 from flask import current_app as app
 
 from auslib.AUS import FORCE_FALLBACK_MAPPING, FORCE_MAIN_MAPPING
 from auslib.web.public.helpers import AUS, get_aus_metadata_headers, get_content_signature_headers, with_transaction
 
+# Map of products and version.
+# Uses legacy key if requested version is < value set here.
+LEGACY_PRODUCT_VERSION_MAPPING = {"FirefoxVPN": "2.22.0"}
+
+
+def check_for_legacy_key(parameters):
```

Comment:
a couple of nits here:
- `use_legacy_key` would be a better name (`check` sounds like it should return nothing, to me)
- you're only using `product` and `version` from parameters - just pass those in rather than everything (it makes it more clear what using a legacy key or not depends on)